mysql-test/suite/rpl/t/percona_bug_ps6979_showprocesslist.test

```diff
@@ -0,0 +1,16 @@
+--source include/master-slave.inc
+
+connection master;
+create database bug111;
```

Comment:
is this statement required?

Reply:
The issue on ps.threads only happens from the second DDL forward.

Reply:
Okay